ariba/2.14.6/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ubuntu:bionic as app
```

Comment:
Is there any reason to continue to use bionic? (There might be dependency issues that I'm not aware of by using jammy, so this is something I'm curious about and I likely won't make you change it)

Reply:
The original Dockerfile for `2.14.4` by @kapsakcj  used `ubuntu:bionic` and worked fine. Just trying to make minimum changes to update the image to `2.14.6`. Any benefit updating to Jammy?

Reply:
We generally prefer to use more recent Ubuntu releases, e.g. jammy and focal, but bionic should continue to receive extended support & security updates through 2028 https://ubuntu.com/about/release-cycle

It's probably not worth the hassle to upgrade to jammy or focal as it may break some of the dependencies installed via `apt-get` since those packages names can change slightly between ubuntu releases.

I'm fine with leaving as ubuntu:bionic, but if a simple upgrade to ubuntu:jammy is possible without too much frustration, I would say let's upgrade.